OpenCL/m27200_a0-pure.cl

```diff
@@ -0,0 +1,134 @@
+/**
+ * Author......: See docs/credits.txt
+ * License.....: MIT
+ */
+
+//#define NEW_SIMD_CODE
+
+#ifdef KERNEL_STATIC
+#include "inc_vendor.h"
+#include "inc_types.h"
+#include "inc_platform.cl"
+#include "inc_common.cl"
+#include "inc_rp.h"
+#include "inc_rp.cl"
+#include "inc_scalar.cl"
+#include "inc_hash_sha1.cl"
+#endif
+
+KERNEL_FQ void m27200_mxx (KERN_ATTR_RULES ())
+{
+  /**
+   * modifier
+   */
+
+  const u64 lid = get_local_id (0);
+  const u64 gid = get_global_id (0);
+
+  if (gid >= gid_max) return;
+
+  const u32 dash[16] = { 0x2d2d0000, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
+
+  /**
+   * base
+   */
+
+  COPY_PW (pws[gid]);
+
+  sha1_ctx_t ctx0;
+
+  sha1_init (&ctx0);
+
+  sha1_update (&ctx0, dash, 2);
```

Comment:
The first entry, where len is guaranteed to by zero, should be initialized with this, since it's static:

```
  ctx0.w0[0] = dash[0];
  ctx0.w0[2] = dash[1];

  ctx.len = 8;
```

Reply:
Does this apply to all the pure kernels in both single-hash and multi-hash modes?

Reply:
Exactly